backend/app/controllers/api/v1/participant/studies_controller.rb

```diff
@@ -20,10 +15,15 @@ def index
   end
 
   def show
-    model =
-      current_user.launched_studies.where(study_id: params[:id]).first ||
-      Study.available_to_participants.find(params[:id])
-    raise ActiveRecord::RecordNotFound if model.is_hidden?
+    model = participant_studies.find do |study|
```

Comment:
Wanted to keep this method in sync with what the list would return (ie this method should only ever be filtering from a list of whats shown to them on the dashboard 🤔 ) hence these changes